GitCommands/AsyncLoader.cs

```diff
@@ -73,13 +73,11 @@ public async Task LoadAsync(Action<CancellationToken> loadContent, Action onLoad
                 // Bail early if cancelled, returning default value for type
                 if (token.IsCancellationRequested)
                 {
-                    result = default;
-                }
-                else
-                {
-                    // Load content
-                    result = loadContent(token);
+                    return default;
```

Comment:
This is the key change. Now, if cancellation is requested, we just return a default value here. Previously we would still return a default value, but call `onLoaded` as well with that default value.

Reply:
But this changes the behavior, previously the callers could detect the cancel scenario. In FileViewer for instance, no text would be shown, now an empty string will be displayed.
I believe the change is fine anyway.

Reply:
I cannot see the behaviour change in FileViewer. Can you help me to see it? All the callbacks I see performed a no-op when passed null.